db/define_schema.rb

```diff
@@ -8,5 +8,10 @@
 DB.define_table("todos")
 DB.define_column("todos", "title", "string")
 DB.define_column("todos", "description", "text")
+DB.define_column("todos", "category_id", "integer")
+DB.define_column("todos", "created_by", "text")
```

Comment:
`DB.define_column("todos", "created_by", "text")` should be `DB.define_column("todos", "created_by", "integer")`, because the purpose of the column is to store the ID of the user who created the todo.